turbopack/crates/turbopack-ecmascript-runtime/js/src/nodejs/runtime.ts

```diff
@@ -177,6 +177,12 @@ function loadWebAssemblyModule(chunkPath: ChunkPath) {
   return compileWebAssemblyFromPath(resolved);
 }
 
+function getWorkerBlobURL(chunks: ChunkPath[]) {
+  let bootstrap = `require(${chunks.map(c => path.resolve(RUNTIME_ROOT, c)).join(", ")});`;
+  let blob = new Blob([bootstrap], { type: "text/javascript" });
+  return URL.createObjectURL(blob);
```

Comment:
We probably want to throw an error here. This is no correct and not implemented (yet).